src/lightning_app/runners/cloud.py

```diff
@@ -197,6 +198,11 @@ def dispatch(
             flow_servers=frontend_specs,
             desired_state=V1LightningappInstanceState.RUNNING,
             env=v1_env_vars,
+            user_requested_flow_compute_config=V1UserRequestedFlowComputeConfig(
```

Comment:
There are other arguments in the CloudCompute class which we are not using here. Perhaps raise error or a warning if those are set. Like the user is specifying a `cluster` in the compute config but is pointed to run on a different cluster etc

Reply:
this was in the code at some point, but as raised [here](https://github.com/Lightning-AI/lightning/pull/14831#discussion_r993556833) it was removed due to the PR that deletes the properties.